R/test_getting_started.R

```diff
@@ -24,7 +24,8 @@ test_getting_started <- function(skip = NULL, ctx = get_default_context()) {
     #' \code{make_context()} must be called before calling any of the
     #' \code{test_} functions in this package}
     has_context = function() {
-      expect_is(ctx, "DBItest_context")
+      object <- ctx # object duplication fixes lazyeval error
```

Comment:
In what context does the error occur, what is the message? Is it perhaps enough to call `expect_is( (ctx), ...)`?